### PR TITLE
fix input fields and scroll bug on edge

### LINKED ISSUE #1877: Change style of not editable input fields
Inputfields which are not editable should only be displayed as text.

### PR DESCRIPTION
**Pull request by JohannesDoberer**
Fixes #1877 
..and scrollbar bug on edge

### CODE REVIEW

File: core/src/UserSettingsEditor.html [hunk 1]
```diff
@@ -1,210 +1,219 @@
...
+    line-height: var(--sapContent_LineHeight, 1.4);
+    font-family: var(--sapFontFamily, "72", "72full", Arial, Helvetica, sans-serif);
+    font-weight: 400;
+    -webkit-font-smoothing: antialiased;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    border: 0;
+    color: var(--sapTextColor, #32363a);
+  }
```
Comment (UlianaMunich) [inline core/src/UserSettingsEditor.html:124]:
```suggestion

```

Comment (UlianaMunich) [reply core/src/UserSettingsEditor.html:124]: I couldn't find where it is used in the file

Comment (JohannesDoberer) [reply core/src/UserSettingsEditor.html]: 👍  thanks. left over

**Review by UlianaMunich — APPROVED** [review]
LGTM

**Review by stanleychh — APPROVED** [review]

**Pull request merged**